examples/consul-examples-helper/consul-examples-helper.sh

```diff
@@ -152,16 +152,20 @@ function get_consul_cluster_ips {
 
 function print_instructions {
   local readonly server_ips=($@)
-  local readonly server_ip="${server_ips[0]}"
+  local readonly server_ip_0="${server_ips[0]}"
+  local readonly server_ip_1="${server_ips[1]}"
+  local readonly server_ip_2="${server_ips[2]}"
```

Comment:
Why this change?

Reply:
To verify that consul has elected a leader and servers have synchronized.